src/pytest_html/nextgen.py

```diff
@@ -243,6 +243,9 @@ def pytest_runtest_logreport(self, report):
             config=self._config, report=report
         )
 
+        if "node" in data:
+            data["node"] = str(data["node"])
```

Comment:
i would like to recommend putting something like

```python
{"gateway": { "id" : node.gateway.id, "spec": node.gateway.spec}}
``` 
but its perhaps best to upstream that asap

until then the str is probably fine